modules/user_manual/pages/apps/admin_audit.adoc

```diff
@@ -0,0 +1,198 @@
+= The Admin Audit App
+:toc: right
+:toclevels: 1
+:leveloffset: +1
+
+== Introduction
+
+The Admin Audit app is an auditing module for ownCloud to trace the actions of users and administrators.
+
+== Configuration
+
+Configuration is needed to redirect the messages into a separate file.
+
+Add these lines to "config.php" and adjust the target path accordingly:
+
+[source,console]
+----
+'log.conditions' => [
+  [
+    'apps' => ['admin_audit'],
+    'logfile' => '/var/www/owncloud/data/admin_audit.log'
+  ]
+]
+----
+
+Please note that the target path must be writeable for the web server user.
+All messages regardless of log level will be logged there.
+To ignore all CLI triggered events (not the default), you can set the following option:
+
+[source,console]
+----
+occ config:app:set admin_audit ignore_cli_events --value='yes'
+----
+
+=== Grouped Logging
+
+With each log message, a number of users are calculated to be the 'audit context'.
+This is the list of users which are related to the log message.
+Additionally, each log message includes a list of groups that the users are a member of, to enable filtering and splitting of the log messages at a later date.
+In cases when users are members of many groups, to reduce the data output, the group list can be filtered using the following config option:
+
+[source,console]
+----
+'admin_audit.groups' => [
+  'group1',
+  'group2'
+]
+----
+
+When the filter is configured, only the filtered list of groups will be output in `auditGroups`, else, all groups that the `auditUsers` are a member of are output.
+
+=== Connect with Splunk Cloud
+
+To connect ownCloud with Splunk Cloud, you need to carry out the steps below:
```

Comment:
add a brief description why you want to do that including a link (https://www.splunk.com)

Reply:
I've done this in the latest commit.